mongomock/collection.py

```diff
@@ -2024,9 +2034,17 @@ def __exit__(self, exc_type, exc_val, exc_tb):
     def alive(self):
         return self._emitted != len(self._compute_results(with_limit_and_skip=False))
 
-    @property
-    def collation(self):
-        return self._collation
+    def collation(self, collation: Collation | Mapping[str, Any]) -> "Cursor":
+        """Adds a :class:`~pymongo.collation.Collation` to this query.
+        Raises :exc:`TypeError` if `collation` is not an instance of
```

Comment:
The summary line of a multi-line docstring has to be separated with a blank line

https://peps.python.org/pep-0257/#multi-line-docstrings